src/typeMapper.js

```diff
@@ -86,6 +86,7 @@ export function toGraphQL(sequelizeType, sequelizeTypes) {
 
   if (sequelizeType instanceof ENUM) {
     return new GraphQLEnumType({
+      name: 'tempEnumName', //grpahql 0.9 requires name for EnumType. Using tempEnumName until its replaced with field generated name
```

Comment:
the name is replaced after type mapper is called so there is no problem for now, it is not the best solution  but maybe you can suggest better